packages/extension-helpers/meta.yaml

```diff
@@ -0,0 +1,9 @@
+package:
```

Comment:
`extension-helpers` is only a build-time dependency. I think it is better just to add this as `pip install extension-helpers` in the build script.

Reply:
At least I believe this is the case...

Reply:
Indeed, thanks! We should really add build dependencies to `meta.yaml` #1476